lib/aptible/api/operation.rb

```diff
@@ -57,25 +57,44 @@ def with_ssh_cmd(private_key_file)
         connection = create_ssh_portal_connection!(ssh_public_key: public_key)
         certificate = connection.ssh_certificate_body
 
-        with_temporary_id(private_key, public_key, certificate) do |id_file|
-          cmd = [
-            'ssh',
-            "#{connection.ssh_user}@#{account.bastion_host}",
-            '-p', account.ssh_portal_port.to_s,
-            '-i', id_file,
-            '-o', 'IdentitiesOnly=yes'
-          ]
-
-          # If we aren't allowed to create a pty, then we shouldn't try to
-          # allocate once, or we'll get an awkward error.
-          cmd << '-T' unless connection.ssh_pty
-
-          yield cmd, connection
+        stack = account.stack
+        host = stack.ssh_portal_host
+        port = stack.ssh_portal_port
+        key = stack.ssh_host_rsa_public_key
+
+        with_temporary_known_hosts(host, port, key) do |known_hosts_file|
+          with_temporary_id(private_key, public_key, certificate) do |id_file|
+            cmd = [
+              'ssh',
+              "#{connection.ssh_user}@#{host}",
+              '-p', port.to_s,
+              '-i', id_file,
+              '-o', 'IdentitiesOnly=yes',
+              '-o', "UserKnownHostsFile=#{known_hosts_file}",
+              '-o', 'StrictHostKeyChecking=yes',
+              '-o', 'UpdateHostKeys=no'
```

Comment:
Good catch; thanks @fancyremarker! I've removed that option, since it wasn't really *required* and there's no reason to unconditionally break the SSH bits of the CLI on older platforms just for the sake of it 😄